src/java/org/apache/cassandra/auth/AllowAllAuthenticator.java

```diff
@@ -22,13 +22,17 @@
 import java.util.Map;
 import java.util.Set;
 
+import com.google.common.collect.Sets;
+
 import org.apache.cassandra.exceptions.AuthenticationException;
 import org.apache.cassandra.exceptions.ConfigurationException;
 
 public class AllowAllAuthenticator implements IAuthenticator
 {
     private static final SaslNegotiator AUTHENTICATOR_INSTANCE = new Negotiator();
 
+    private static final Set<String> MODES = Sets.newHashSet(AuthenticatedUser.ANONYMOUS_USERNAME);
```

Comment:
it feels strange to me to reuse "anonymous user name" for something like authentication modes. I know that "well it is 'anonymous' and we need that constant' but we should not reuse that even it is of same value.

Reply:
:+1:, makes sense.  Would 'unauthenticated' be ok in this case?  Either that or 'none'?

Reply:
went with 'Unauthenticated' but if have a suggestion for a name would be happy to change it :+1:

Reply:
that's fine